include/sys/zfs_rlock.h

```diff
@@ -60,8 +70,12 @@ typedef struct rl {
  * or exclusive (RL_WRITER or RL_APPEND).  RL_APPEND is a special type that
  * is converted to RL_WRITER that specified to lock from the start of the
  * end of file.  Returns the range lock structure.
+ *
+ * Filesystem should call zfs_range_lock.
+ * Zvol should call zvol_range_lock.
```

Comment:
This comment should be removed, there is no `zvol_range_lock()` and `zfs_range_lock()` should be usable by all dmu consumers.